api/build/create.go

```diff
@@ -133,15 +139,14 @@ func CreateBuild(c *gin.Context) {
 	)
 
 	// check if build was skipped
-	if err != nil && strings.EqualFold(item.Build.GetStatus(), constants.StatusSkipped) {
+	if err != nil && code == http.StatusOK {
 		c.JSON(http.StatusOK, err.Error())
 
 		return
 	}
 
-	// error handling done in CompileAndPublish
 	if err != nil {
-		return
+		util.HandleError(c, code, err)
```

Comment:
do we still need the return here? so we dont double write to the response

Reply:
Yes